src/reducers/versions.spec.tsx

```diff
@@ -1832,6 +1835,32 @@ describe(__filename, () => {
       );
     });
 
+    it('appends showDiff if requested', async () => {
+      const location = createFakeLocation({ pathname });
+      const history = createFakeHistory({ location });
+      const diffPosition = ScrollTarget.firstDiff;
+
+      const { dispatch, thunk } = thunkTester({
+        createThunk: () =>
+          viewVersionFile({
```

Comment:
Just a personal preference but anytime an arrow function goes multiline, I like to see it wrapped in curly braces. This makes it easier to avoid a syntax error when you go to edit the return value later on.